packages/dds/sequence/src/revertibles.ts

```diff
@@ -176,6 +208,14 @@ function addIfIntervalEndpoint(
 	return false;
 }
 
+/**
+ * Create revertibles for SharedStringDeltas
```

Comment:
Might also be good to clarify that this is an edit to the string that subsequently affects the interval.